examples/examples_test.go

```diff
@@ -388,41 +388,6 @@ func TestMigrateRdsInstance(t *testing.T) {
 ]`)
 }
 
-func TestRegressUnknownTags(t *testing.T) {
```

Comment:
Should this stay? This was testing against panics when unknowns are passed into tags.

Reply:
It should. I've added it back.